webapp/src/main/java/life/qbic/email/Email.java

```diff
@@ -1,7 +1,7 @@
 package life.qbic.email;
 
 /**
- * A simple representation of an Email
+ * A simple representation of an EmailAddress
```

Comment:
The class is still called `Email` and in this case, I think the replacement went crazy again. Please change back to `Email`
```suggestion
 * A simple representation of an Email
```